ansible/roles/cli_vault/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Add an Apt signing key for Hashicorp
+  ansible.builtin.apt_key:
+    url: https://apt.releases.hashicorp.com/gpg
+    state: present
+  become: true
+
+# TODO: Find a way to retrieve this information from the system one time
+- name: Found and register "$VERSION_CODENAME"
+  ansible.builtin.shell:
+    cmd: ". /etc/os-release && echo ${VERSION_CODENAME}"
+  become: true
```

Comment:
Isn't the os-release file readable by all users by default?

Reply:
i don't think so